src/m_isotopes.f90

```diff
@@ -0,0 +1,37 @@
+module m_isotopes
```

Comment:
For consistency with other module names, could you please change `m_isotopes` to `isotopes_module`?

Reply:
I suggest changing the name of the file to `isotopes.f90`.